src/components/TextField/TextField.style.ts

```diff
@@ -1,32 +1,85 @@
 import { css, SerializedStyles } from '@emotion/core';
-import { rem, transparentize } from 'polished';
-import { Theme } from '../../theme';
 import { Props } from './TextField';
+import { rem } from 'polished';
 
+const wrapperStyleSwitch = (theme: Theme, lean?: boolean, error?: boolean, styleType?: string) => {
+  if (lean) {
```

Comment:
Maybe adding `lean` to the styling options, should be a good idea.
Since I am not aware of which projects are using the lean version of the component, my suggestion is to add it to the component's `styleType` available options and add a todo to remove the `lean` property when we have replaced it with `styleType="lean"` to our projects!

Reply:
lean is to bypass anything that styles the input as a general but you want the mechanism - check datepicker